packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -87,38 +60,34 @@ const _ActionList = ({
   // 1. We don't render the box wrapper styles which includes shadows, padding, border etc
   // 2. to ensure GorhomBottomSheetSectionList works as expected, if we add extra wrappers GorhomBottomSheet won't render the content inside
   // NOTE: That this also means inside BottomSheet, ActionList won't render any ActionListHeader or Footer.
-  return (
-    <ActionListContext.Provider value={actionListContextValue}>
```

Comment:
Removed the provider?

Reply:
yes. It was only used for passing `surfaceLevel` prop down. That prop is removed and dropdown has static non-configurable color now